galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager/src/main/java/dev/galasa/zos/internal/ZosManagerImpl.java

```
@@ -221,6 +222,16 @@ public void provisionGenerate() throws ManagerException, ResourceUnavailableExce
             dependencyTags.remove("PRIMARY");
         }
 
+        System.out.println("LEWO");
+        System.out.println(cps.getAllProperties());
+        try {
+			System.out.println(cps.getProperty("image", "ports", "MV2D"));
+		} catch (ConfigurationPropertyStoreException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();

```

need to re-throw an exception